.github/workflows/notify.yml

```diff
@@ -0,0 +1,19 @@
+name: Notify CPT
+on:
+  pull_request:
+    types: [opened, reopened]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: send telegram message on push
+        uses: appleboy/telegram-action@master
+        with:
+          to: ${{ secrets.TELEGRAM_TO }}
+          token: ${{ secrets.TELEGRAM_TOKEN }}
```

Comment:
Could you put these in the one password? I think it would be good practice to keep those there and up to date